.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,8 +8,6 @@ Fixes # (issue)
  Loom Video: https://www.loom.com/
 -->
 
-**Environment**: Staging(main branch) / Production
```

Comment:
Discussed earlier in group that this isn't needed as we do patch releases every other day and main is almost the production.